module/x/gravity/types/genesis.go

```diff
@@ -440,6 +450,13 @@ func validateEthereumBlacklistAddresses(i interface{}) error {
 	return nil
 }
 
+func validateMinChainFeeBasisPoints(i interface{}) error {
+	if _, ok := i.(uint64); !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
```

Comment:
we should check the value is less than 100% here. Just becuase anything crazy large set by accident will hit that one.

Reply:
Done